core/src/main/cpp/gdn/gast_node.cpp

```diff
@@ -301,13 +119,16 @@ int GastNode::get_external_texture_id(int surface_index) {
         surface_index = kDefaultSurfaceIndex;
     }
 
-    ExternalTexture *external_texture = get_external_texture(surface_index);
-    int tex_id = external_texture == nullptr ? kInvalidTexId
+    Ref<ExternalTexture> external_texture = get_external_texture(surface_index);
+    int tex_id = external_texture.ptr() == nullptr ? kInvalidTexId
```

Comment:
You can use `external_texture.is_null()` instead.

Reply:
Done.